flokk_src/lib/_internal/components/pinned_stack.dart

```diff
@@ -21,8 +20,8 @@ class PinnedStack extends StatelessWidget {
   final Clip clip;
 
   const PinnedStack(
-      {Key key,
-      this.children,
+      {Key? key,
+      this.children = const <Widget>[],
```

Comment:
required instead? More helpful when using it, to be reminded if we forget to pass children